src/ar65/library.c

```diff
@@ -36,6 +36,12 @@
 #include <stdio.h>
 #include <string.h>
 #include <errno.h>
+#if defined(_WIN32)
+#include <process.h>
+#else
+#include <sys/types.h>
+#include <unistd.h>
+#endif
```

Comment:
Some nitpicking? ;-)
The source location I pointed you to uses `#  include` (note the two blanks) for "ifdef'ed" include statements.